sql/messaging_system_derived/cfr_users_last_seen_v1/query.sql

```diff
@@ -0,0 +1,42 @@
+WITH _current AS (
+  SELECT
+-- In this raw table, we capture the history of activity over the past
+-- 28 days as a single 64-bit integer. The rightmost bit represents
+-- whether the user was active in the current day.
+    CAST(TRUE AS INT64) AS days_seen_bits,
+    * EXCEPT (submission_date)
+  FROM
+    cfr_users_daily_v1
```

Comment:
I guess this should be the raw table, i.e. `messaging_system_stable.cfr_v1` or `cfr_v1` for short here?

Reply:
Oh, I see. You created a view called `cfr_users_daily` above.

Reply:
Hmm, I am still a bit confused here about the name. `cfr_users_daily_v1` should be the table name, correct?

Reply:
See https://github.com/mozilla/bigquery-etl#views

> Views should generally not be named with a version suffix; a view represents a stable interface for users and whenever possible should maintain compatibility with existing queries; if the view logic cannot be adapted to changes in underlying tables, breaking changes must be communicated to fx-data-dev@mozilla.org

So the raw table has a version, but the view does not.